lms/templates/edxnotes/toggle_notes.html

```diff
@@ -8,12 +8,13 @@
   edxnotes_visibility_url = reverse("edxnotes_visibility", kwargs={"course_id": course.id})
 %>
 <div class="wrapper-utility edx-notes-visibility">
+  <span class="action-toggle-message" aria-hidden="true">${_("Hiding Notes")}</span>
```

Comment:
@cptvitamin, quick question on this new UI element (meant to communicate system status of showing/hiding notes). I gave it an ``aria-hidden`` attribute just to avoid it mis-communicating state, but if you think this is useful and not redundant, we can use an ``aria-live`` and show/speak it.